scripts/sdk-version-bump.py

```
@@ -0,0 +1,388 @@
+
+def regenerate_lockfile(runner_dir: Path) -> bool:
+    """Run `uv lock` to regenerate the lockfile.
+
+    Args:
+        runner_dir: Path to the runner component directory.
+
+    Returns:
+        True if successful.
+    """
+    print("  Running uv lock...")
+    result = subprocess.run(
```

Ruff (0.15.6)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)